htdocs/edittags.bml

```diff
@@ -103,8 +103,11 @@ body<=
     $ret .= "<script type='text/javascript'> var cur_taglist = '$logtagstr'; </script>";
 
     $ret .= '<table summary="" class="edittbl" cellpadding="0" cellspacing="0" width="50%">';
+    if ( $u->is_community ) {
+        $ret .= "<tr><td class='l'>$ML{'.poster'}</td><td>" . $ent->poster->ljuser_display . "</td></tr>"
+        };
     $ret .= "<tr><td class='l'>$ML{'.subject'}</td><td>$subj</td></tr>" if $subj;
-
+    
```

Comment:
Since I'm already requesting a change to adhere to our coding style guidelines, can also I ask you to take out the extra whitespace that was added here?